cmd/save.go

```diff
@@ -31,7 +29,7 @@ var saveCmd = &cobra.Command{
 	Short: "save changes to a dataset",
 	Long: `
 Save is how you change a dataset, updating one or more of data, metadata, and 
-structure. Every time you run save, an entry is added to your dataset’s log 
+structure. You can also update your data via url. Every time you run save, an entry is added to your dataset’s log 
```

Comment:
These strings are tuned to use an 80 character ruler for cleaner CLI output. Sublime text has view > ruler > 80 to put a ruler in yo editor.

Reply:
omg thank you I've been stressing about this